packages/@aws-cdk/aws-kms/README.md

```diff
@@ -16,30 +16,26 @@ key.addAlias('alias/foo');
 key.addAlias('alias/bar');
 ```
 
-### Importing and exporting keys
+### Sharing keys between stacks
 
-To use a KMS key that is not defined within this stack, use the
-`EncryptionKey.import(parent, name, ref)` factory method:
+To use a KMS key in a different stack in the same CDK application,
+pass the construct to the other stack:
```

Comment:
It's almost so simple now that it's not worth calling it out. We don't say, "to use a KMS key with your bucket, pass the key in the Bucket props". We just list the properties.

Reply:
I know, but we used to recommend something else, so I want to make it explicitly obvious what the new recommendation is.